src/main/java/com/playtika/homework4/methods/Methods.java

```
@@ -0,0 +1,234 @@
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.util.EntityUtils;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+
+public class Methods {
+
+
+    public static CreatedUser createUser() throws IOException {
+        System.out.println("Введите e-mail");
+        Scanner sc = new Scanner(System.in);
+        String email = sc.nextLine();
+
+        System.out.println("\nВведите пароль");
+        String password = sc.nextLine();
+        try {
+            CloseableHttpClient httpClient = HttpClients.createDefault();
+            HttpPost httpPost = new HttpPost("https://taschool-notes-service.herokuapp.com/v1/accounts");
+            httpPost.setHeader("Content-Type", "application/json");
+
+            ObjectMapper mapper = new ObjectMapper();
+            ObjectNode node = mapper.createObjectNode();
+            node.put("email", email);
+            node.put("password", password);
+            String JSON_STRING = node.toString();
+            httpPost.setEntity(new StringEntity(JSON_STRING, ContentType.APPLICATION_JSON));
+            HttpResponse response = httpClient.execute(httpPost);
+            HttpEntity entity = response.getEntity();
+            String stringEntity = EntityUtils.toString(entity);
+            CreatedUser createdUser = mapper.readValue(stringEntity, CreatedUser.class);
+
+            System.out.println("\nПользователь: " + "'" + createdUser.getEmail() + "'" + " с ID " + "'" + createdUser.getId() + "'" + " успешно создан");
+            return createdUser;
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (InputMismatchException e) {
+            System.out.println("Вы ввели некорректное значение. Попробуйте еще раз.");
+            return createUser();
+        }
+        return null;
```

instead of returning null just throw exception - in all methods of this class